go-controller/pkg/cni/utils.go

```diff
@@ -133,6 +133,7 @@ func PodAnnotation2PodInfo(podAnnotation map[string]string, checkExtIDs bool, po
 }
 
 //Taken from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/types/pod_update.go
+//Start
```

Comment:
move this to the previous commit:  `[Add Functions needed to check for static pod](https://github.com/ovn-org/ovn-kubernetes/pull/3347/commits/d94386ae0542a8a2d088d34aa5d231d09dd0c4fc)`